MAUI/Cartesian-Charts/Zooming-and-panning.md

```diff
@@ -248,4 +248,131 @@ chart.ZoomPanBehavior = zooming;
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Selection zooming
+
+Selection zooming feature allows users to interactively choose a particular area of the chart and zoom in. By specifying [EnableSelectionZooming]() property to `true` as shown in the following code snippet, you can double tap and drag to select a range on the chart to be zoomed in. The default value of this property is false.
+
+{% tabs %}
```

Comment:
@EzhilarasanElangovan31 Mention how to perform selection zooming on interaction.

Reply:
mentioned how to perform selection zooming on interaction